mlfinlab/tests/test_run_data_structures.py

```diff
@@ -105,7 +105,7 @@ def test_run_tick_bars(self):
         """
         Tests the imbalance tick bars implementation.
```

Comment:
Should this rather be run bars? it mentions imbalance. Check this in the other doc strings as well.